### ISSUE TITLE
Lora fine-tunning Error：binascii.Error: Invalid base64-encoded string: number of data characters (85) cannot be 1 more than a multiple of 4

### ISSUE THREAD

**Issue by jjyu-ustc** [ymcui/Chinese-LLaMA-Alpaca#620]
### 提交前必须检查以下项目

- [X] 请确保使用的是仓库最新代码（git pull），一些问题已被解决和修复。
- [X] 由于相关依赖频繁更新，请确保按照[Wiki](https://github.com/ymcui/Chinese-LLaMA-Alpaca/wiki)中的相关步骤执行
- [X] 我已阅读[FAQ章节](https://github.com/ymcui/Chinese-LLaMA-Alpaca/wiki/常见问题)并且已在Issue中对问题进行了搜索，没有找到相似问题和解决方案
- [X] 第三方插件问题：例如[llama.cpp](https://github.com/ggerganov/llama.cpp)、[text-generation-webui](https://github.com/oobabooga/text-generation-webui)、[LlamaChat](https://github.com/alexrozanski/LlamaChat)等，同时建议到对应的项目中查找解决方案
- [X] 模型正确性检查：务必检查模型的[SHA256.md](https://github.com/ymcui/Chinese-LLaMA-Alpaca/blob/main/SHA256.md)，模型不对的情况下无法保证效果和正常运行

### 问题类型

模型训练与精调

### 基础模型

Alpaca-13B

### 操作系统

Linux

### 详细描述问题

Hello, everyone!
Thanks for any answer that may help.

I am trying to fine-tune Alpaca-Chinese-13B with "run_sft.sh"
But I just don't get it how to set "peft_model" in training Alpaca-Chinese-13B. (Not a plus version)
The conda environment is 100% excellent cause I just used it to train Alpaca-Chinese-13B-Plus.

Am I right to set the variables like this:

pretrained_model=/root/autodl-tmp/Alpaca/13B/path_to_original_llama_hf_dir (which is the hf version of llama-13b)
chinese_tokenizer_path=/root/autodl-tmp/Alpaca/13B/chinese_alpaca_lora_13b
peft_model=/root/autodl-tmp/lora13b (which is the merged model of alpaca-chinese and llama-hf )

<img width="572" alt="image" src="https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/108610753/bb5a3bd0-5e7a-4493-91e3-6ac402250236">

Unfortunately, something rather unexpected came out as a result. 

<img width="577" alt="image" src="https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/108610753/bf583d43-c337-4584-adce-c8055b7af2e0">

Apologize for any inconvenience that may cause. Any help or reply is appreciated. 
Best regards.

**Comment by airaria** [comment]
You should set `peft_model` to the lora model (`chinese-alpaca-lora-13b`), not the merged model.
Besides, the error looks like being related to deepspeed. Did you modify the deepspeed config file?

**Comment by JingJieYu0** [comment]
Thank you very much for your kind reply.
I modified the peft_model, but now the problem is still unsolved.
Even I tried to fine-tune a chinese-alpaca-13b-PLUS using bash like this 

<img width="574" alt="image" src="https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/108610753/3bce8b04-6930-4e13-88e1-0d29aa0ce6cb">

It is quite strange that the code works on chinese-alpaca-7b-PLUS, but it doesn't work on chinese-alpaca-13b-PLUS suggesting errors in deepspeed. I reinstalled deepspeed, but it doesn't work.

Thank you again for your reply.
